pkg/converter/convert_unix.go

```diff
@@ -712,41 +726,61 @@ func MergeLayers(ctx context.Context, cs content.Store, descs []ocispec.Descript
 	buffer := bufPool.Get().(*[]byte)
 	defer bufPool.Put(buffer)
 	if _, err := io.CopyBuffer(compressed, pr, *buffer); err != nil {
-		return nil, errors.Wrapf(err, "copy bootstrap targz into content store")
+		return nil, nil, errors.Wrapf(err, "copy bootstrap targz into content store")
 	}
 	if err := gw.Close(); err != nil {
-		return nil, errors.Wrap(err, "close gzip writer")
+		return nil, nil, errors.Wrap(err, "close gzip writer")
```

Comment:
Not related to this PR. But `gw` can be nil. We should handle the exception.
In addition, `gw` should also be closed when returning from L729

Reply:
![image](https://user-images.githubusercontent.com/1524576/191231932-c53163c2-e8b1-4df0-b2e4-e282972a88d5.png)

It's ok if `gw.Close` not be called when returning from L729.

Reply:
Then where does it flush the data?

Reply:
If an error happens, no data need to be flushed.